src/main/resources/liquibase/changelog.xml

```diff
@@ -37,5 +37,6 @@
     <include file="liquibase/202011130945-add-events-table.xml" />
     <include file="liquibase/202101081600-tweak-offering.xml" />
     <include file="liquibase/202101121122-add-subscription-table.xml" />
+    <include file="liquibase/202102031030-tweak-subscription.xml" />
```

Comment:
Can we give this a more descriptive name? `update-subscription-table-pkey.xml`

Reply:
Done. I'll push this with my next commit to the branch.